frontend/src/pages/CourseSelector/CourseTabs/CourseTabs.jsx

```diff
@@ -40,6 +42,16 @@ const CourseTabs = () => {
 
   return (
     <S.CourseTabsWrapper>
+      <S.ShowAllCourses>
+        <Switch
+          size="small"
+          defaultChecked={showLockedCourses}
+          onChange={() => dispatch(toggleLockedCourses())}
+        />
+        <S.TextShowCourses>
```

Comment:
Also, text should be before (left to) the toggle switch as per the ticket description. *i.e. something like this **(DISPLAY LABEL TO LEFT OF TOGGLE INSTEAD)**:*